cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -973,22 +972,33 @@ toAlonzoPParamsUpdate
     , Alonzo._minPoolCost     = toShelleyLovelace <$>
                                   maybeToStrictMaybe protocolUpdateMinPoolCost
     , Alonzo._coinsPerUTxOWord  = toShelleyLovelace <$>
-                                  maybeToStrictMaybe protocolUpdateUTxOCostPerWord
+                                  maybeToStrictMaybe
+                                    (protocolUpdateUTxOCostPerWord >>= non 0)
```

Comment:
I don't understand what is going on here, or why. Zero is a perfectly acceptable value of the protocol param. Setting the param to zero is not the same as not changing the param it at all.

Reply:
ok, removed